tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/mobile@2021_02_12/bugly.pysnap

```diff
@@ -100,13 +100,13 @@ app-depth-max:
               "kill"
             package (ignored because function takes precedence)
               "libc.so"
-        type (ignored because exception is synthetic (detected via exception type))
+        type (ignored because exception is synthetic)
           "SIGSEGV"
         value (ignored because stacktrace takes precedence)
           "Segfault"
 --------------------------------------------------------------------------
 system:
-  hash: "d9c9b0f9ba46e32fddd7cd1512fad235"
+  hash: "5d36f5363b3d4125907c15b93f85bbd8"
```

Comment:
@untitaker Just to double-check: The system hash changes because it now ignores the `synthetic` marker. This is what we wanted, right?

Reply:
that's okay as long as the app hash does not change